documentation/1.9/content/userguide/tools/rebase-image.md

```diff
@@ -19,9 +19,10 @@ Usage: imagetool rebase [OPTIONS]
 | --- | --- | --- |
 | `--additionalBuildCommands` | Path to a file with additional build commands. For more details, see [Additional information](#additional-information). |
 | `--additionalBuildFiles` | Additional files that are required by your `additionalBuildCommands`.  A comma separated list of files that should be copied to the build context. |
+| `--builder`, `-b` | Executable to process the Dockerfile. Use the full path of the executable if not on your path. | `docker`  |
```

Comment:
Use the full path of the executable if not on your path -> If not on your path, then use the full path of the executable.